.dockerignore

```diff
@@ -0,0 +1,3 @@
+node_modules
+bundle-3003.js
+bundle-3003.js.map
```

Comment:
I don't think we need bundle-3003 in here anymore?

We could probably add bundle-review.js*

Reply:
Also, let's add:

.DS_Store

Reply:
oh yea thanx for checking 
i will modify this file as well!

Reply:
you can remove the .DS_Store file that was added with:
git rm

Reply:
Also the bundles, can we remove those with git rm too?